pkg/remote/remote.go

```diff
@@ -168,6 +176,9 @@ func (r *Remote) Fetch(goGetterSrc string, cacheDirOpt ...string) (string, error
 	// e.g. os.CacheDir()/helmfile/https_github_com_cloudposse_helmfiles_git.ref=0.xx.0
 	cacheDirPath := filepath.Join(r.Home, getterDst)
 
+	// e.g. os.CacheDir()/helmfile/https_github_com_cloudposse_helmfiles_git.ref=0.xx.0/origin
+	originDirOrFilePath := filepath.Join(cacheDirPath, "origin")
```

Comment:
Why do we need this `origin` component? @yxxhero

Reply:
Work in progress.